src/renderRepoCard.js

```diff
@@ -44,7 +44,7 @@ const renderRepoCard = (repo, options = {}) => {
   const multiLineDescription = wrapTextMultiline(desc);
   const descriptionLines = multiLineDescription.length;
   const lineHeight = 10;
-
+  const isGradient = !(bg_color == undefined || bg_color.length == 6 || bg_color.length == 3)
```

Comment:
I used this algorithm to detect gradients. It's not the best (detects by length), so any ideas to improve it are appreciated.

Reply:
Outdated now, so resolving conversation...